shell/platform/embedder/embedder.cc

```diff
@@ -1126,6 +1126,15 @@ FlutterEngineResult FlutterEngineInitialize(size_t version,
     }
   }
 
+  if (SAFE_ACCESS(args, dart_entrypoint_argc, 0) > 0 &&
```

Comment:
If the embedder provides more than 0 arguments but fails to provide a non-null `dart_entrypoint_argv`, then you can abort engine initialization with kInvalidArgs and an error (using the `LOG_EMBEDDER_ERROR` macro used in this TU) instead of silently ignoring the error.